app/assets/javascripts/support.js

```diff
@@ -1,6 +1,6 @@
 if (typeof window.GOVUK === 'undefined') { window.GOVUK = {} }
 if (typeof window.GOVUK.support === 'undefined') { window.GOVUK.support = {} }
 
-window.GOVUK.support.history = function () {
+window.GOVUK.support.history = function history () {
```

Comment:
Does this overwrite the `window.history` method?

Reply:
_Is such a thing even possible?_

Reply:
I couldn't get `window.history` or `history` to be overwritten - even when I tried to ([Codepen sketch](https://codepen.io/injms/pen/BeQEeP?editors=1111)). MDN says it's a read-only property, so I think that this is okay.

Reply:
Good stuff, thanks for checking!